website_twitter_no_ext_links/models/twitter_theme.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class TwitterTheme(models.Model):
+    _name = 'twitter.theme'
+    
+
+    name = fields.Char(
+        string= 'Twitter Theme Name',
+        required=True,
+    )
+
+    def _get_colors(self):
```

Comment:
depend on https://github.com/OCA/web/tree/8.0/web_widget_color and use that instead